### PR TITLE
docs: use mockbin to view http logs for guidance

### PR DESCRIPTION
**Pull request by Yiyiyimu**
Signed-off-by: yiyiyimu <wosoyoung@gmail.com>

### What this PR does / why we need it:
In slack, people are asking
> hello, i want to use http logger plugin, but i don't understand what should i put in the uri , and how to view the logs?

So try to make docs runnable for beginners.

### Pre-submission checklist:

* [ ] Did you explain what problem does this PR solve? Or what new features have been added?
* [ ] Have you added corresponding test cases?
* [ ] Have you modified the corresponding document?
* [ ] Is this PR backward compatible? **If it is not backward compatible, please discuss on the [mailing list](https://github.com/apache/apisix/tree/master#community) first**

### CODE REVIEW

File: docs/en/latest/plugins/http-logger.md [hunk 1]
```diff
@@ -54,14 +54,14 @@ This will provide the ability to send Log data requests as JSON objects to Monit
 
 ## How To Enable
 
-The following is an example on how to enable the http-logger for a specific route.
+The following is an example on how to enable the http-logger for a specific route. You could generate an mock HTTP server at [mockbin](http://mockbin.org/bin/create) to view the logs.
```
Comment (spacewander) [inline docs/en/latest/plugins/http-logger.md:57]:
```suggestion
The following is an example of how to enable the http-logger for a specific route. You could generate a mock HTTP server at [mockbin](http://mockbin.org/bin/create) to view the logs.
```

Comment (tzssangglass) [inline docs/en/latest/plugins/http-logger.md:57]: enable the `http-logger` for

Comment (Yiyiyimu) [reply docs/en/latest/plugins/http-logger.md]: Sorry I didn't get what you mean

**Review by spacewander — APPROVED** [review]

File: docs/en/latest/plugins/http-logger.md (hunk 1 above)
Comment (tzssangglass) [reply docs/en/latest/plugins/http-logger.md]: add ``  around the http-logger

File: docs/en/latest/plugins/http-logger.md [hunk 2]
```diff
@@ -54,14 +54,14 @@ This will provide the ability to send Log data requests as JSON objects to Monit
 
 ## How To Enable
 
-The following is an example on how to enable the http-logger for a specific route.
+The following is an example of how to enable the http-logger for a specific route. You could generate a mock HTTP server at [mockbin](http://mockbin.org/bin/create) to view the logs.
```
Comment (Yiyiyimu) [inline docs/en/latest/plugins/http-logger.md:57]:
```suggestion
The following is an example of how to enable the `http-logger` for a specific route. You could generate a mock HTTP server at [mockbin](http://mockbin.org/bin/create) to view the logs.
```

File: docs/zh/latest/plugins/http-logger.md [hunk 3]
```diff
@@ -54,14 +54,14 @@ title: http-logger
 
 ## 如何开启
 
-这是有关如何为特定路由启用 http-logger 插件的示例。
+这是有关如何为特定路由启用 http-logger 插件的示例。你可以在 [mockbin](http://mockbin.org/bin/create) 生成一个模拟 HTTP 服务器来浏览生成的日志
```
Comment (Yiyiyimu) [inline docs/zh/latest/plugins/http-logger.md:57]:
```suggestion
这是有关如何为特定路由启用 `http-logger` 插件的示例。你可以在 [mockbin](http://mockbin.org/bin/create) 生成一个模拟 HTTP 服务器来浏览生成的日志
```

File: docs/en/latest/plugins/http-logger.md (hunk 1 above)
Comment (Yiyiyimu) [reply docs/en/latest/plugins/http-logger.md]: Thanks! fixed

**Review by tzssangglass — APPROVED** [review]

**Review by membphis — APPROVED** [review]
nice PR

**Pull request merged**